hyp3_testing/templates/autorift_golden.json.j2

```diff
@@ -88,5 +88,45 @@
        ]
     },
     "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LC09_L1GT_215109_20220125_20220125_02_T2",
+        "LC09_L1GT_215109_20220210_20220210_02_T2"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LE07_L1TP_063018_20040911_20200915_02_T1",
+        "LE07_L1TP_063018_20040810_20200915_02_T1"
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT05",
+        "LT05"
```

Comment:
@forrestfwilliams @jacquelynsmale can you add one of the Landsat 5 pairs you've been working with to add L5 support?

Reply:
We were working with one scene:  LT05_L1TP_018013_20060610_20200901_02_T1 so far. I think LT05_L1TP_018013_20060626_20200831_02_T1 or LT05_L1GS_018013_20060407_20200901_02_T2 could be a good secondary or reference scene.

Reply:
Oddly enough, this pair wasn't in the pair list provided to us by ITS_LIVE. I went back through the pair list and selected a pair with <5% cloud cover in Greenland. The pair name is `'LT05L1GS0010131992062801T2_LT05L1GS0010131992042501T2_32624_G0240V01_P008.nc'`, so the scenes are `T05_L1GS_001013_19920628_20200914_02_T2` and `LT05_L1GS_001013_19920425_20200915_02_T2`